app/templates/application.hbs

```diff
@@ -46,11 +46,18 @@
             Wissel van bestuurseenheid
           </AuLink>
 
-          {{#unless this.currentSession.showAdminFeatures}}
+          {{#if
+            (not
```

Comment:
again another place where you repeat the same logic, but now in a different way of writing it down. If we ever update this we will never find this. please keep the getter